templates/static/privacy.html

```diff
@@ -0,0 +1,217 @@
+{% extends 'layouts/_base.html' %}
+{% import 'macros/helpers.html' as helpers %}
+
+{% set survey_title = survey_title or _("ONS Business Surveys") %}
+{% set use_transactional_footer = true %}
+
+{% block main %}
+  {% do page | setAttribute("feedback", "") %}
+
+  <h1>{{ _("Privacy and data protection") }}</h1>
+  <p>{{ _("The information you provide when you fill in your census rehearsal questionnaire is protected by law.") }}</p>
+  <p>{{ _("Here we detail why we need your information. We also describe all the steps that we take to make sure your
+  information and your privacy are safe.") }}</p>
+
+  <br>
+  <h2>{{ _("How and why we use your information") }}</h2>
+
+  <p>{{ _("We collect information from your questionnaire under our statutory objective to promote and
+  safeguard the production of official statistics that serve the public good. This means that any personal data we collect will only
+  ever be used to produce statistics or undertake statistical research. People will not be identified within those statistics or research.") }}</p>
+
+  <p>{{ _("Every 10 years, the census gathers vital information that helps the government and local authorities
+  plan services like health services, roads and libraries.") }}</p>
+
+  <p>{{ _("By running a rehearsal for Census 2021, we can test our processes, systems and services and make
+  sure everything goes smoothly for the real thing. The personal data you provide on your questionnaire will also be used to undertake
+  research and produce statistics that serve the public good, both by the Office for National Statistics (ONS) and accredited
+  researchers from other organisations.") }}</p>
+
+  <p>{{ _("We may occasionally record information about a property or an individual, in the interests of safeguarding the wellbeing
+  of both those being interviewed and interviewers, as well as to assist people in the completion of their questionnaires. We’ll hold the minimum
+  amount of personal data needed to meet these purposes.") }}</p>
+
+  <p>{{ _("For online completion, we’ll collect information about how people fill in their questionnaires online to provide us with
+  knowledge of how to improve our online services.") }}</p>
+
+  <br>
+  <h2>{{ _("Who can access the information?") }}</h2>
+
+  <p>{{ _("During the rehearsal we’ll employ third-party providers to assist us with parts of the operation.
+  These third parties work under our instruction.") }}</p>
+  <p>{{ _("We treat the information we hold with respect, keeping it secure and confidential. Data may be made available to approved
+  researchers where there is a clear public value and it’s safe and lawful to do so. More information on") }}
+  <a href="https://www.ons.gov.uk/aboutus/transparencyandgovernance/lookingafterandusingdataforpublicbenefit/policies">
+  {{ _("how we allow access to data for research") }}</a> {{ _("is available. We also publish a full list of all the") }}
```

Comment:
Strings shouldn't be split as it makes it very difficult to translate them. For text with links we should use the form:
```python
{{ _("This <a href='{url}'>link</a> is helpful").format(url='http://www.google.com') }}
```